pkg/solidity-utils/test/foundry/utils/BaseTest.sol

```diff
@@ -31,6 +31,8 @@ abstract contract BaseTest is Test, GasSnapshot {
     bytes32 internal constant ZERO_BYTES32 = 0x0000000000000000000000000000000000000000000000000000000000000000;
     bytes32 internal constant ONE_BYTES32 = 0x0000000000000000000000000000000000000000000000000000000000000001;
 
+    address internal constant ZERO_ADDRESS = address(0);
```

Comment:
Nice!

Reply:
Yes, sorry for introducing an extraneous thing, but this is an obvious thing to define, which was being defined at the test level in a few other places.